### PR TITLE
MAINT Refactor *console.py*

### PR DESCRIPTION
**Pull request by UltraStudioLTD**
> Changes

 - Make `_banner` function's return value inline and remove function

### Description

I converted what `_banner` function in **console.py** did into one line string, assigned it to `BANNER` variable and removed function with it's *"deleter"* (`del _banner` https://github.com/UltraStudioLTD/pyodide/commit/71962ed927dbd73804d84ba148bfe9ea810abc8a#diff-9fadbc85698015f27704ee7dabbb14b811a184a2907a910ef792f892138b1182L43)

### CODE REVIEW

File: src/py/_pyodide/console.py [hunk 1]
```diff
@@ -30,17 +30,7 @@
...
-    # copied from https://github.com/python/cpython/blob/799f8489d418b7f9207d333eac38214931bd7dcc/Lib/code.py#L214
-    cprt = 'Type "help", "copyright", "credits" or "license" for more information.'
-    version = platform.python_version()
-    build = f"({', '.join(platform.python_build())})"
-    return f"Python {version} {build} on WebAssembly VM\n{cprt}"
-
-
-BANNER = _banner()
-del _banner
+BANNER = f"Python {platform.python_version()} ({', '.join(platform.python_build())}) on WebAssembly VM\nType \"help\", \"copyright\", \"credits\" or \"license\" for more information."
```
Comment (ryanking13) [inline src/py/_pyodide/console.py:33]: For little more readability, maybe:

```python
from platform import python_version, python_build

BANNER = f""" 
Python {python_version()} ({', '.join(python_build())}) on WebAssembly VM
Type "help", "copyright", "credits" or "license" for more information.
""".strip()
```

**Review by ryanking13 — COMMENTED** [review]
Thanks, @UltraStudioLTD!

Please keep the original comment for future reference,
and update the line number in the test.

https://github.com/pyodide/pyodide/blob/75aa9ba678e0a8733259fea3cc3e4c786e12fab7/src/tests/test_console.py#L432

File: src/py/_pyodide/console.py [hunk 2]
```diff
@@ -29,9 +29,10 @@
 
 __all__ = ["repr_shorten", "BANNER", "Console", "PyodideConsole", "ConsoleFuture"]
 
-
-BANNER = f"Python {platform.python_version()} ({', '.join(platform.python_build())}) on WebAssembly VM\nType \"help\", \"copyright\", \"credits\" or \"license\" for more information."
```
Comment (UltraStudioLTD) [inline src/py/_pyodide/console.py]: Removed old version

File: src/py/_pyodide/console.py [hunk 3]
```diff
@@ -29,9 +29,10 @@
...
-
+BANNER = f""" 
+Python {python_version()} ({', '.join(python_build())}) on WebAssembly VM
+Type "help", "copyright", "credits" or "license" for more information.
+""".strip()
```
Comment (UltraStudioLTD) [inline src/py/_pyodide/console.py:36]: New expanded version

File: src/py/_pyodide/console.py [hunk 4]
```diff
@@ -10,7 +10,7 @@
 )
 from contextlib import _RedirectStream  # type: ignore
 import rlcompleter
-import platform
```
Comment (UltraStudioLTD) [inline src/py/_pyodide/console.py:13]: Remove `platform`'s global import

File: src/py/_pyodide/console.py [hunk 5]
```diff
@@ -10,7 +10,7 @@
...
+from platform import python_version, python_build
```
Comment (UltraStudioLTD) [inline src/py/_pyodide/console.py:13]: Only import what we need from `platform`

**Comment by ryanking13** [comment]
Please run `black` to adjust formatting, and the test is still failing due to the line number. The line number should match the `text = repr(value)` line (It's at line 477 now, but it could be changed after formatting).

https://github.com/pyodide/pyodide/blob/0a425e99a3837c5aa4dbcddd2322871492ec4c61/src/py/_pyodide/console.py#L477

**Comment by UltraStudioLTD** [comment]
I ran `black` but it says that no files were changed

**Comment by hoodmane** [comment]
What's your black version? Should be:
```sh
$ black --version
black, 21.12b0 (compiled: no)
```

**Comment by hoodmane** [comment]
What's your black version? Should be:
```sh
$ black --version
black, 21.12b0 (compiled: no)
```

**Comment by hoodmane** [comment]
What's your black version? Should be:
```sh
$ black --version
black, 21.12b0 (compiled: no)
```

**Comment by UltraStudioLTD** [comment]
Just upgraded to `black, 22.1.0 (compiled: no)`

**Comment by UltraStudioLTD** [comment]
Wait a minute

**Comment by ryanking13** [comment]
> Just upgraded to `black, 22.1.0 (compiled: no)`

It looks like black `22.1.0` updated the formatting rule compare to the version `21.12b0` that we are using in CI.
For this PR, please use downgrade it to `21.12b0` then run it again. Except for that LGTM. Sorry to bother!

**Comment by UltraStudioLTD** [comment]
> 

Alright

**Comment by hoodmane** [comment]
Thanks @UltraStudioLTD!

**Pull request merged**